beetmoverscript/templates/thunderbird_candidates.yml

```diff
@@ -104,6 +104,10 @@ mapping:
       s3_key: beetmover-checksums/source/thunderbird-{{ version }}.checksums.asc
       destinations:
         - beetmover-checksums/source/thunderbird-{{ version }}.checksums.asc
+    buildhub.json:
```

Comment:
This is harmless in the templates so we can definitely keep this. However, we'd need to double-check with @tomprince that `buildhub.json` is in the upstreamArtifacts for Thunderbird jobs. Otherwise, it won't update anything. Looking at the latest Thunderbird beta, sounds like it's not https://tools.taskcluster.net/groups/RTCkxBCKTYeozYKGn5ZYSg/tasks/JybFCvuJSFOmPRe-JBibLw/details but I strongly suspect the changes for buildhub.json were riding the trains so the newly generated betas on Thunderbird should have it. We need to talk to Tom for this. It'd be nice to fix this for Thunderbird as well since we're here.